core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/test/mocks.ts

```diff
@@ -97,3 +88,28 @@ export class ActivatedRouteMock {
         };
     }
 }
+
+export class ActivatedRouteListStoreMock {
+    get snapshot() {
+        return {
+            params: {
+                pageId: '1111'
+            },
+            parent: {
+                parent: {
+                    parent: {
+                        parent: {
+                            data: {
+                                content: {
+                                    page: {
+                                        title: 'title'
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        };
+    }
+}
```

Comment:
I know this may be needed for the `dot-experiment` project and we should plan a meeting so we can talk about it.